packages/php/compat-checker/src/Checks/WCCompatibility.php

```diff
@@ -0,0 +1,389 @@
+				'</a>'
+			);
+
+			$this->add_admin_notice(
+				'woocommerce-not-installed',
+				'error',
+				$message
+			);
+		}
+	}
+
+	/**
+	 * Add notices for out of date WooCommerce.
+	 */
+	public function wc_out_of_date() {
+		if ( ! current_user_can( 'update_plugins' ) ) {
+			return;
+		}
+
+		$plugin_name         = $this->plugin_data['Name'];
+		$wc_version_required = $this->plugin_data['RequiresWC'];
+
+		$update_url = wp_nonce_url( self_admin_url( 'update.php?action=upgrade-plugin&plugin=' . self::WC_PLUGIN_FILE ), 'upgrade-plugin_' . self::WC_PLUGIN_FILE );
+		$message    = sprintf(
+			/* translators: %1$s - Plugin Name, %2$s - minimum WooCommerce version, %3$s - update WooCommerce link open, %4$s - update WooCommerce link close, %5$s - download minimum WooCommerce link open, %6$s - download minimum WooCommerce link close. */
+			esc_html__( '%1$s requires WooCommerce version %2$s or higher. Please %3$supdate WooCommerce%4$s to the latest version, or %5$sdownload the minimum required version &raquo;%6$s', 'woogrow-compat-checker' ),
+			'<strong>' . $plugin_name . '</strong>',
+			$wc_version_required,
+			'<a href="' . esc_url( $update_url ) . '">',
+			'</a>',
+			'<a href="' . esc_url( 'https://downloads.wordpress.org/plugin/woocommerce.' . $wc_version_required . '.zip' ) . '">',
+			'</a>'
+		);
+
+		$this->add_admin_notice(
+			'woocommerce-out-of-date',
+			'error',
+			$message
+		);
+	}
+
+	/**
+	 * Adds notice for untested WooCommerce.
+	 */
+	public function wc_untested() {
+		$plugin_name    = $this->plugin_data['Name'];
+		$wc_version     = $this->get_wc_version();
+		$plugin_version = $this->plugin_data['Version'];
+
+		$message = sprintf(
+			/* translators: %1$s - Plugin Name, %2$s - Plugin version, %3$s - WooCommerce version number */
+			esc_html__( '%1$s - %2$s is untested with WooCommerce %3$s.', 'woogrow-compat-checker' ),
+			'<strong>' . $plugin_name . '</strong>',
+			$plugin_version,
+			$wc_version
+		);
+
+		$this->add_admin_notice(
+			'woocommerce-untested',
+			'warning',
+			$message
+		);
+	}
+
+	/**
+	 * Adds notice for WooCommerce upgrade recommendation.
+	 */
+	public function make_upgrade_recommendation() {
+		// Bail if the user does not have the update plugins capability.
+		if ( ! current_user_can( 'update_plugins' ) ) {
+			return;
+		}
+
+		$plugin_name        = $this->plugin_data['Name'];
+		$current_wc_version = $this->get_wc_version();
+
+		$message = sprintf(
+			/* translators: Placeholders: %1$s - plugin name, %2$s - WooCommerce version number, %3$s - opening <a> HTML link tag, %4$s - closing </a> HTML link tag */
+			esc_html__( 'Heads up! %1$s will soon discontinue support for WooCommerce %2$s. Please %3$supdate WooCommerce%4$s to take advantage of the latest updates and features.', 'woogrow-compat-checker' ),
+			'<strong>' . $plugin_name . '</strong>',
+			$current_wc_version,
+			'<a href="' . esc_url( admin_url( 'update-core.php' ) ) . '">',
```

Comment:
This URL is better, it just takes the merchant to the screen where they can decide whether to update WooCommerce or not. 
💅🏻  Maybe add `#update-plugins-table` to the URL to jump straight to upgradeable Plugins?

Reply:
Thank you for this suggestion. I've added the anchor jump to the URL fragment in this commit: https://github.com/woocommerce/grow/pull/79/commits/394a108d78dcbb8f3c77b6861d0b6a2828ffd320